src/Illuminate/Queue/Worker.php

```diff
@@ -231,14 +254,22 @@ public function runNextJob($connectionName, $queue, WorkerOptions $options)
      */
     protected function getNextJob($connection, $queue)
     {
+        $this->output->writeln('Trying to fetch a job...', OutputInterface::VERBOSITY_VERBOSE);
+
         try {
             foreach (explode(',', $queue) as $queue) {
                 if (! is_null($job = $connection->pop($queue))) {
+                    $preview = str_limit(json_encode($job->payload(), 256));
+                    $this->output->writeln("Successfully fetched a job:\n{$preview}\n", OutputInterface::VERBOSITY_VERY_VERBOSE);
```

Comment:
Please consider using `PHP_EOL` constant instead of `\n`, cos different platform use various line endings.